packages/edit-site/src/components/page-patterns/index.js

```diff
@@ -65,6 +66,9 @@ const { ExperimentalBlockEditorProvider, useGlobalStyle } = unlock(
 const templatePartIcons = { header, footer, uncategorized };
 const EMPTY_ARRAY = [];
 const defaultConfigPerViewType = {
+	[ LAYOUT_TABLE ]: {
+		primaryField: 'title',
```

Comment:
What's the primary field of a "table" layout? What's different from regular fields?

Reply:
For [styling](https://github.com/WordPress/gutenberg/blob/trunk/packages/dataviews/src/style.scss#L237) and announcing [selection](https://github.com/WordPress/gutenberg/blob/trunk/packages/dataviews/src/single-selection-checkbox.js#L24).